sql/CreateScript.sql

```diff
@@ -0,0 +1,82 @@
+-- create database
+CREATE DATABASE IF NOT EXISTS linkedin;
+
+-- use created database
+USE linkedin;
+
+-- create users table
+CREATE TABLE IF NOT EXISTS users(
+  id int  AUTO_INCREMENT PRIMARY KEY,
+  email varchar(50) UNIQUE,
```

Comment:
You can add "username" as a Unique attribute as well. since two users can't have the same username at the signup.

Reply:
Username is not required during Linkedin registration, so we can only depend on the email for uniquely identifying the user (at least for now)